### PR TITLE
Section header

### LINKED ISSUE #1524: <SectionHeader />
```js
<SectionHeader
  title={?string}
  link={?Component}
/>
```

screen shot 2016-12-01 at 3 14 04 pm

### PR DESCRIPTION
**Pull request by JakeDawkins**
# Section Header
resolves #1524 

Added a section header with a heading on the left, and a component (button) on the right

@samclaridge what is the padding around this supposed to be? It looks like soft-half-quarter on the ends (not what I have in  the screenshot below). And the primary-dark here looks darker than the screenshot. It looks more like dark-tertiary

_note:_ the button is currently broken. btn--dark-primary shows the default green.

screen shot 2016-12-04 at 5 25 19 pm

Yours:

### CODE REVIEW

**Comment by jbaxleyiii** [comment]
**[Storybook Hub](https://hub.getstorybook.io/)** is building storybooks for every commit in this PR.

**Latest Storybook: https://2a6a1963-b155-4e19-8856-6fb35e153509.sbook.io/**
<sub>( Visit [here](https://hub.getstorybook.io/app/github.com/NewSpring/Holtzman/branch/section-header/) for more info )</sub>

> _This comment is automatically added by Storybook Hub_

**Comment by samclaridge** [comment]
@JakeDawkins that's my bad, dark-primary was from a previous iteration. You're right the correct text color is dark-tertiary. Where you apply the padding is going to depend on how you structure the markup. The section header text should have padding left of 20px, or soft. The button small should have margin right of 10px. I imagined that those would be floating in a containing div that has 20px padding on the ends.

https://cl.ly/3C3A3H073V1u

File: imports/components/sectionHeader/index.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+
+const halvesClasses = "one-half floating display-inline-block soft-half";
+
+type ISectionHeader = {
+  title?: string,
+  link?: Object, // React component
+};
+
+const SectionHeader = ({ title, link }: ISectionHeader) =>
+  <div className="background--light-secondary one-whole">
```
Comment (jbaxleyiii) [inline imports/components/sectionHeader/index.js]: @JakeDawkins let's leave the coloring to the parent component, so you can remove the background color here

**Comment by JakeDawkins** [comment]
Updated. I removed the bg color, and fixed the padding 

screen shot 2016-12-05 at 8 56 16 am
screen shot 2016-12-05 at 8 56 41 am
screen shot 2016-12-05 at 8 56 54 am
screen shot 2016-12-05 at 8 57 14 am

**Pull request merged**